src/tempfile.rs

```diff
@@ -72,13 +72,17 @@ impl NamedTempfile {
     pub(super) fn file(&self) -> &File {
```

Comment:
Let's change this method, to `fn take_file() -> Option<File>`, so we don't need the `close()` method